### DISCUSSION TITLE
speedtest-monitoring/

### DISCUSSION THREAD

**Discussion by giscus** [netdata/blog#146] [General]
# speedtest-monitoring/

Monitoring and troubleshooting ISP performance.

https://blog.netdata.cloud/speedtest-monitoring/

**Comment by potistiri** [comment]
Great Article. The problem with ookla speedtest cli is that you cannot define different countries than yours. by default speedtest bring you the top 5 or 10 closest to your area and in most cases that's only to your country. From GUI is way much different.

**Comment by ghost** [comment]
Unfortunately as per this thread is has developed some issues that need resolving otherwise I was looking extremely excited to trying this: https://community.netdata.cloud/t/speedtest/3642
